edx-olive/configuration-old
# SE-1021 Adds asgard role and other changes to deploy jenkins_admin

Updates required to run the `edx-east/vpc_admin.yml` playbook.

* Adds asgard (`hotg`) role to configuration repo, cherry-picked from upstream
* Pulls latest `jenkins_admin` role from upstream master.
* Disables the checks and tasks which back up Jenkins to S3 -- we don't need this.

**Testing instructions**

Used to redeploy the new admin instance.  Verify that asgard is running and contains these entries in the instance type list: `t3.xlarge`, `c5.xlarge`.

**Author Notes & Concerns**

1. Changes to the asgard (`hotg`) role are submitted upstream with https://github.com/edx/configuration/pull/5348.

**Reviewer**
